e2e/tests/v2Manage.spec.ts

```diff
@@ -50,3 +54,56 @@ test('Future manager marks a bed as out of service in the V2 Manage area', async
   // And I see the success message on the 'history' pane of the bed page
   await revisitedV2BedPage.showsOutOfServiceBedRecordedSuccessMessage()
 })
+
+test('Future manager updates an out of service bed', async ({ page, futureManager }) => {
+  // Given I am signed in as a future manager
+  await signIn(page, futureManager)
+
+  // And I am on the list of premises page
+  const dashboard = await visitDashboard(page)
+  await dashboard.clickManage()
+  const premisesListPage = await PremisesListPage.initialize(page, 'List of Approved Premises')
+
+  // When choose to view the detail of a particular premises
+  await premisesListPage.choosePremises(premisesName)
+
+  // Then I should see the premises page
+  const premisesPage = await PremisesPage.initialize(page, premisesName)
+
+  // When I choose to manage its out of service beds
+  await premisesPage.viewOutOfServiceBedRecords()
+
+  // Then I should see the out of service beds list page for the premises
+  const manageOOSBedsPage = await OutOfServiceBedsPremisesListPage.initialize(page)
+
+  // When I select the 'future' tab and select the out of service bed created earlier
+  await manageOOSBedsPage.selectFutureTab()
+  await manageOOSBedsPage.selectOutOfServiceBed()
+
+  // Then I should see the out of service bed page
+  const outOfServiceBedPage = await OutOfServiceBedPage.initialize(page)
+
+  // When I select 'Update record'
+  await outOfServiceBedPage.selectUpdateRecord()
+
+  // Then I should see the update out of service bed record page
+  const updateOOSBedPage = await UpdateOutOfServiceBedPage.initialize(page)
+
+  // When I update the out of service bed record
+  const update = {
+    referenceNumber: faker.string.uuid(),
+    additionalInformation: faker.lorem.sentence(),
```

Comment:
would you agree that this test will be easier to debug if we use realistic content for the update?

e.g.

```ts
const update = {
  referenceNumber: 'REF-ABC123',
  additionalInformation: 'The supplier informed us of a delay in starting work'
}
```

In that way an error message might say 

> expected to find 'The supplier informed us of a delay in starting work' on the page

rather than

> expect to find 'Lorem ipsum nonsense gobbledeegook' on the page

Reply:
I do agree. But having a unique string allows us to cover the case where the same update has been made in E2E tests multiple times. Then we can assert that each unique string is present on the timeline page, without having to write a more complex function to target the specific element we're looking for. To me, I thought it was worth trading off the time it would take to do that against a slightly less intuitive debugging experience. But maybe not 🤷

Reply:
> I do agree. But having a unique string allows us to cover the case where the same update has been made in E2E tests multiple times. Then we can assert that each unique string is present on the timeline page, without having to write a more complex function to target the specific element we're looking for.

ah interesting, I hadn't understood that. How about combining the approaches?

e.g.

```ts
const uniqueUpdateId = faker.string.uuid()
const update = {
  referenceNumber: `UPDATED-REF [${uniqueUpdateId}]`,
  additionalInformation: `Additional info on update [${uniqueUpdateId}]`
}
```

Reply:
ok, good idea!